docs/development/howtocontributewebsite.md

```diff
@@ -7,49 +7,52 @@ group: development
 
 # Contributing to Apache Zeppelin ( Website )
 
-## IMPORTANT
-Apache Zeppelin is an [Apache2 License](http://www.apache.org/licenses/LICENSE-2.0.html) Software.
-Any contribution to Zeppelin (Source code, Documents, Image, Website) means you agree license all your contributions as Apache2 License.
+<div id="toc"></div>
 
 
-## Modifying the website
+This page will give you an overview of how to build and contribute to the documentation of Apache Zeppelin.
+The online documentation at [zeppelin.apache.org](https://zeppelin.apache.org/docs/latest/) is also generated from the files found here.
 
-#### Getting the source code
-Website is hosted in 'master' branch under `/docs/` dir.
+> **NOTE :** Apache Zeppelin is an [Apache2 License](http://www.apache.org/licenses/LICENSE-2.0.html) Software.
+Any contributions to Zeppelin (Source code, Documents, Image, Website) means you agree with license all your contributions as Apache2 License.
 
-First of all, you need the website source code. The official location of mirror for Zeppelin is [http://git.apache.org/zeppelin.git](http://git.apache.org/zeppelin.git).
+## Getting the source code
+First of all, you need Zeppelin source code. The official location of Zeppelin is [http://git.apache.org/zeppelin.git](http://git.apache.org/zeppelin.git).
+Documentation website is hosted in 'master' branch under `/docs/` dir.
+
+### git access
 
+First of all, you need the website source code. The official location of mirror for Zeppelin is [http://git.apache.org/zeppelin.git](http://git.apache.org/zeppelin.git).
```

Comment:
I think git.apache.org is the actual source code, not a "mirror", but we are told not to have github.com address here.